plugin/.eslintrc.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": ["@alleyinteractive/eslint-config/typescript-react"]
+}
```

If files are resolved with Webpack you will need to add the following configuration:
```suggestion
{
  "extends": ["@alleyinteractive/eslint-config/typescript-react"],
  "settings": {
    "import/resolver": "webpack"
  }
}

```